src/libsync/syncfilestatustracker.cpp

```diff
@@ -157,6 +162,20 @@ SyncFileStatus SyncFileStatusTracker::fileStatus(const QString& systemFileName)
     return SyncFileStatus();
 }
 
+void SyncFileStatusTracker::slotPathTouched(const QString& systemFileName)
+{
+    QString fileName = systemFileName.normalized(QString::NormalizationForm_C);
```

Comment:
Is that even required?  I think the file system watcher already make sure that the file names have the right normalisation.